packages/react-core/src/components/Modal/ModalBox.tsx

```diff
@@ -7,10 +7,8 @@ export interface ModalBoxProps extends React.HTMLProps<HTMLDivElement> {
   children: React.ReactNode;
   /** Additional classes added to the ModalBox */
   className?: string;
-  /** Creates a large version of the ModalBox */
-  isLarge?: boolean;
-  /** Creates a small version of the ModalBox. */
-  isSmall?: boolean;
+  /** Variant of the modal */
+  variant?: 'large' | 'medium' | 'small';
```

Comment:
Isn't what you have here as `medium` the default which is full width?  Maybe call it `full` or `default`?
cc @mcoker

Reply:
I'll change it to `default`